rdflib/namespace/__init__.py

```diff
@@ -76,31 +91,9 @@
     "Namespace",
     "ClosedNamespace",
     "NamespaceManager",
-    "CSVW",
-    "DC",
-    "DCAT",
-    "DCTERMS",
-    "DOAP",
-    "FOAF",
-    "ODRL2",
-    "ORG",
-    "OWL",
-    "PROF",
-    "PROV",
-    "QB",
-    "RDF",
-    "RDFS",
-    "SDO",
-    "SH",
-    "SKOS",
-    "SOSA",
-    "SSN",
-    "TIME",
-    "VOID",
-    "XMLNS",
-    "XSD",
```

Comment:
Should not have been.  Probably a cut and paste error.

Reply:
Actually, they weren't.  `namespace.py` no longer exists -- instead becoming `namespace/__init__.py`

Reply:
Ah yes, I see. However, the classes were lifted to modules, so anyone who imported `rdflib.namespace.TIME` will get a nasty surprise when `TIME` is suddenly a module rather than a class. I suspect that this means that the files will need to be renamed to avoid collisions in the `rdflib.namespace` (python) namespace.

Reply:
Actually, it should work as expected, as the module is named _TIME and it is lifted in rdflib.namespace.   In general, if there are nasty surprises waiting, shouldn't there be tests for them?   I'd been running under the assumption that if something matters there is (or will be)  a unit test for it -- is this not the case?

Reply:
Apologies I must have brainfarted hard on the file names or been fooled by the fact that github didn't update them in the interface. I see that this was already addressed by 91015c19d10229137bdf81a47bcae9d864d1983d, so before I made the previous comment, and addresses exactly the nasty surprise that I was imagining. Whether there were explicit tests for the class names, I suspect that there were quite a few implicit tests simply by virtue of having to import them at all.

Reply:
Yup - one of the take-home lessons I've learned from this push is that it is far more important to review changes to unit tests than to the code base itself.  I (stupidly) made several changes to the unit tests to make them match the new code, initially hiding the "nasty surprises".  Hopefully I have undone all of those and can somewhat confidently assert that the set of nasty surprises in the unit tests have been covered.